cfpb/consumerfinance.gov
# Global Search: revamp fallback | no-js innerText

Global Search does not work properly in the no-js state (legacy IE mostly). This PR simplifies the state by providing a fallback link to the search page instead of trying to override the interactive search box state.

Also, Firefox 44 and earlier doesn't support `innerText`, so we want to deliver the no-js state to those browsers.

## Changes

- Revamps global search no-js fallback to provide a simpler to maintain link state when JavaScript is disabled.
- Includes lack of support for innerText in site's no-js delivery.


## How to test this PR

1. Pull branch and run locally.
2. Visit the homepage and inspect the markup. In the `<html …` element, change the `js` class to `no-js`.
3. Visit the homepage in Firefox 44 or earlier in Sauce Labs and see it shows the fallback search (you'll need Sauce Connect running to be able to view localhost).

## Screenshot

<img width="1299" alt="Screen Shot 2020-08-28 at 11 09 37 AM" src="https://user-images.githubusercontent.com/704760/91600118-6a0d0c80-e935-11ea-92b7-4781a791b0da.png">
